### DISCUSSION TITLE
Sync not requiring Google account

### DISCUSSION THREAD

**Discussion by Whayme** [runelite/runelite#15482] [Q&A] [answered]
It would be very nice to be able to sync Runelite settings without relying on a privacy-sapping external account.  Maybe generic WebDAV or Nextcloud?

**Comment by Enriath** [comment, marked as answer]
Google is only used as an authentication method; nothing is actually stored on a service controlled by them.

**Reply by Whayme** [reply]
My concern is less about my Runelite data being accessed by Google, and more about the sync feature requiring an account with them.  I'm not comfortable accessing their services or making an account.

**Reply by Enriath** [reply]
That account only has as much data as you give them. Use a fresh email address that's unique, and just give them bogus information if needed.

**Comment by Nightfirecat** [comment]
We may in the future offer this, but as Hydrox said, Google is only used for authentication, not for anything else. Furthermore, you don't need to use the Account plugin to sync your settings, you can use any file-syncing service such as Dropbox. (See: #2494)
